### PR TITLE
Spanish

### PR DESCRIPTION
**Pull request by lozalojo**

### CODE REVIEW

File: df-logger/lang/es.json [hunk 1]
```diff
@@ -43,7 +70,7 @@
 			"21": "Se acerca el Challenger - ¡{{username}} ha aparecido!",
 			"22": "¿Es un pájaro? ¿Es un avión? Nah, olvídalo, solo es {{username}}.",
 			"23": "¡Es {{username}}! ¡Alabado sea el Sol!\n\\\\[T]/",
-			"24": "Nunca perderé la confianza en {{username}}. Nunca decepcionaré a {{username}}.",
+			"24": "{{username}} es un honrado traficante\nTomando copas me lo encuentro to' los días\nMe cuenta historias de sus años en la cárcel\nA veces había suerte y si tenía pasta salía",
```
Comment (flamewave000) [inline df-logger/lang/es.json:73]: I think this is a personal message. Supposed to be a line from the song "Never Gonna Give You Up".

Comment (flamewave000) [inline df-logger/lang/es.json:73]: I think this is a personal message. Supposed to be a line from the song "Never Gonna Give You Up".

**Review by flamewave000 — CHANGES_REQUESTED** [review]

File: df-logger/lang/es.json (hunk 1 above)
Comment (lozalojo) [reply df-logger/lang/es.json:73]: > I think this is a personal message. Supposed to be a line from the song "Never Gonna Give You Up".

I know, it is a Rick Astley song, the problem is that if I directly translate it to spanish, it loses its meaning. I mean, nobody is going to recognize the reference to the song as it is translated to spanish.

I had two options: use the original english text or replace the song for another song, in this case a popular spanish song from Extremoduro, one of the biggest spanish rock bands of all times. I went for the second option.

What do you think? Do you prefer keeping the old Rick?

Comment (flamewave000) [reply df-logger/lang/es.json:73]: It's tough, I definitely like the idea of you using culturally appropriate lyrics, and I'm also a big fan of classic rock. I'm just unsure about the drinking part of the quote you've got. I want to keep it professional and PG-13. Is there maybe a different portion of the song you could use?

Comment (lozalojo) [reply df-logger/lang/es.json:73]: > It's tough, I definitely like the idea of you using culturally appropriate lyrics, and I'm also a big fan of classic rock. I'm just unsure about the drinking part of the quote you've got. I want to keep it professional and PG-13. Is there maybe a different portion of the song you could use?

Yes, it is a song speaking about Pepe Botika (Jonnhy "The pharmacist", in reference to his occupation: a small-time drug dealer) and has some mature content, thou Extremoduro was very popular among teenagers some years ago.

I have to think on another song, but I simply cant stand mainstream music or something PEG.13 people hear nowadays.

Let me think on it.

Comment (lozalojo) [reply df-logger/lang/es.json]: Ops, I forgot to update the PR, I have replaced the song for another one: "Bienvenidos", from "Miguel Ríos", it does not contain any scary parts, it is just an hymn to Rock'n'roll and it is widely known in Spain.

**Pull request closed without merging**